### PR TITLE
Roadmap/Readme Update

### LINKED ISSUE #882: Add any missing full lesson outlines from roadmap spreadsheet to root README
We have added a lot of content, and need help updating our Techtonica/curriculum Readme Table of Contents. https://github.com/Techtonica/curriculum#table-of-contents

Keep the current format, but add any missing or outdated links in the "outline" column in the Techtonica Roadmap to our ReadMe. This includes assessments and tutorial links, and any link in the "outline" column that isn't private access only.
https://docs.google.com/spreadsheets/d/1sGJpzOfaZJW8OedBBf5eHOBppivAoc8Ug4tEmxG8IZ8/edit?usp=sharing

- The RoadMap is the source of truth.  If the order is different in the RoadMap, change the README to match.
- No duplicates, please, even if there are duplicates on the Roadmap.
- add missing full lesson outline files (no homework, no external links, no worksheets, no code challenges)
- Fix dead or incorrect links in Readme.
- Contact @alodahl  or @vegetabill for any questions about content.

### PR DESCRIPTION
**Pull request by abhu-A-J**
fixes #882 

Updated the RoadMap by changing the order of outlines, and removing outlines not present in the [Techtonica Apprenticeship 2020 H1 Roadmap](https://docs.google.com/spreadsheets/d/1sGJpzOfaZJW8OedBBf5eHOBppivAoc8Ug4tEmxG8IZ8)

### CODE REVIEW

**Comment by abhu-A-J** [comment]
@alodahl I have changed the numbering scheme as you suggested. And I could find the following outlines were not present. I'll add more if I find some more. These are just taken from Roadmap itself and not internal links for outlines.

Let me know if any more changes are required, I'll be happy to address those.

The following links are not present in new RoadMap compared to previous one (unused):

- Expectations While at Techtonica
-  Homebrew
- Keyboard Shortcuts
- Growth Mindset
- Asking Good Questions
- Git Part 2: Study Guide
- Table Relationships and Join Queries by Treehouse
- React Part 2
- Deploying to Heroku with React and Node
- Database Migrations
- Resources: Women in Tech
- Financial Literacy
- Difficult Workplace Situations
-Ethics in Software Engineering

**Comment by alodahl** [comment]
Hi @abhu-A-J , all those seem pretty important, so I am going to ask @vegetabill about them.  You are welcome to pick up a second issue while we discuss it.

**Comment by alodahl** [comment]
Oh, but resolve the conflicts before your new issue.

File: README.md [hunk 1]
```diff
@@ -19,183 +19,245 @@ The following is the approximate sequence of the topics during the program but i
 ### Week 0 - Pre-Work
 Before beginning the program, apprentices meet weekly for 4 weeks to learn JavaScript fundamentals.
 
-### Week 1 - Onboarding & Developer Tools
+### Week 1 - Introduction & Developer Tools
```
Comment (alodahl) [inline README.md]: Why did you change this?

Comment (abhu-A-J) [reply README.md]: It's mentioned on the Roadmap Doc as Introduction & Developer Tools rather than onboarding, so I changed it to make it consistent with the RoadMap Doc itself. Should I change it back?

**Comment by abhu-A-J** [comment]
@alodahl Ma'am, I have resolved all the conflicts now, you can discuss about the changes  as necessary. Meanwhile, I will work on some other issue.

**Comment by vegetabill** [comment]
@alodahl as we discussed in Slack re: maintenance burden of this file, can you please edit this issue to only add missing full lesson outline files (no homework, no external links, no worksheets, etc)? We want to keep its footprint as small as possible.

@abhu-A-J thanks for working on this but we need to revise the original issue. We will let you know what it is revised. Sorry for the inconvenience.

**Comment by alodahl** [comment]
@vegetabill im worried about the removed outlines not being anywhere... should they move to electives? I’m also wondering why they are not in the roadmap.

@abhu-A-J thanks for your patience, I’m glad you’re able to work on other issues.

**Comment by vegetabill** [comment]
@alodahl sorry if my comment wasn't clear. Any missing lessons should be added to the roadmap. If they don't fit anywhere, they can at least be listed at the end under "Miscellaneous." Does that provide enough clarity or do you want me to review a list of the missing items?

**Comment by alodahl** [comment]
@vegetabill yes, could you look at the outlines listed here that Abhu did not find in the RoadMap? https://github.com/Techtonica/curriculum/pull/1278#issuecomment-599623307
 I am sort of looking to you as the Roadmap owner, but if you don't have time I can see where to put them in. It looks like they are in order of appearance.  I personally think it should all be kept, but I would understand changing these three if you had a good reason:
Git Part 2: Study Guide
Table Relationships and Join Queries by Treehouse
React Part 2

**Comment by vegetabill** [comment]
@alodahl those links are already in the roadmap. I tried to git blame where they came from but this entire issues appears moot by now, unfortunately. I'm closing it.

**Comment by abhu-A-J** [comment]
@Kundan28 Sir @alodahl  Ma'am will these not be opened in future?? Is this permanently closed??

**Comment by Kundan28** [comment]
> @Kundan28 Sir @alodahl Ma'am will these not be opened in future?? Is this permanently closed??

@vegetabill

**Comment by alodahl** [comment]
Sorry if that was sudden @abhu-A-J , but Bill seemed sure that the issue wasn't necessary anymore.  Please feel free to pick up another issue!
